### DISCUSSION TITLE
Dark/Light Mode

### DISCUSSION THREAD

**Discussion by nibl** [forever-jekyll/forever-jekyll#2] [Ideas]
It would be good if Forever Jekyll supported Dark Mode switching. I'm just working on it for my own site. If anyone is interested I can make a version for general use. The tricky part is that there are lots of different places where text colour needs to be changed. I'm about 95% there.

**Comment by forever-jekyll** [comment]
Hi there @nibl 
Sorry for the late reply. A few real world tragedies prevented me from replying earlier.
I've implemented a theme switcher that will allow visitors to toggle between dark & light themes. You can see it live in action on the demo page and sync/update your local branch with the [forever-jekyll](https://github.com/forever-jekyll/forever-jekyll) main branch to test yourself.
https://forever-jekyll.github.io/

And I'd love to see your implementation/version of this too so please share it.
